### PR TITLE
Rt review2

### PR DESCRIPTION
**Pull request by rtillies**
- Updated instructions in README, marked merge steps as **IMPORTANT**
- Added rubric to README
- Split SQL file so that setup was its own file, students can add queries in a separate file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,23 @@
...
-5. Commit your changes with this commit message: complete questions
-6. Merge your `questions` branch into the `main` branch.
+5. Commit your changes with this commit message: `complete questions`
+6. **IMPORTANT** Merge your `questions` branch into the `main` branch.
 7. Create a branch called `SQL`.
 8. In PgAdmin, create a database called `SpotifyTopSongs`.
-9. Run the SETUP SQL queries in SQL.txt (you can copy everything from lines 3 to 63 and run it all at once).
-10. Create the queries described in the QUERY section in SQL.txt (starting on line 65). Replace each `< YOUR QUERY >` placeholder with the correct query.
+9. Run the SETUP SQL queries in db_setup.txt (you can copy the entire file and run it all at once).
+10. Create the queries in SQL.txt. Replace each `< YOUR QUERY >` placeholder with the correct query.
```
Comment (memcmahon) [inline README.md:17]: Let's update this to the dbSetup.txt file

File: README.md [hunk 2]
```diff
@@ -6,14 +6,23 @@
...
+11. Commit your changes with this commit message: `complete SQL queries`
+12. **IMPORTANT** Merge your `SQL` branch into the main branch.
 
 ## Rubric
+
```
Comment (memcmahon) [inline README.md:22]: Let's include the passing score so that the rubrics mimic what we had in Mod1

**Review by memcmahon — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (rtillies) [reply README.md:22]: Done!

File: README.md (hunk 1 above)
Comment (rtillies) [reply README.md:17]: Done!

**Pull request merged**